webapp/src/LegacyBundle/Controller/FallbackController.php

```diff
@@ -83,6 +83,7 @@ public function fallback(Request $request, $path)
         ob_start();
         global $G_SYMFONY;
         $G_SYMFONY = $this->container->get('domjudge.domjudge');
+        $G_SYMFONY_RENDER = array($this, 'renderView');
```

Comment:
Isn't it better, especially after merging 3.4 which makes more in the container private by default, to inject `Twig_Environment` in the constructor of the FallbackController and setting that as a variable here? That is what `renderView` uses to render the views

Reply:
Yes! that does indeed sound wiser; I was taking the easy way since searching the docs didn't find me that information quickly and I wanted to skip to just having it working.